benchmark/BaseBench.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace MongoDB\Benchmark;
```

Comment:
Is this file missing a copyright block?

Do we actually need those for benchmarks? We don't use it in our test files.

Reply:
We don't use them for tests, so I omitted them here as well.